### PR TITLE
adding a side structure

### LINKED ISSUE #5: Side stepper
Implement a multi-step form layout in React, which includes a sidebar displaying the steps of the form.

- [ ] Create the Form Container
- [ ] Implement Steps Sidebar
- [ ] Create Responsive Layout

### PR DESCRIPTION
**Pull request by ireemalqarni**
closes #5 

implements a multi-step form
1. Created a sidebar to display the steps:
Step 1: Your Info 
Step 2: Select Plan
Step 3: Add-Ons
Step 4: Summary

2. Set the sidebar background to an SVG image.
3. Styled the sidebar text and step indicators.

-Proof of success
Screenshot 2

### CODE REVIEW

**Comment by MaymoonaAlBoloshi** [comment]
still needs work to match the UI,

File: src/pages/page1.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+        <div className="form-content">
+
+<div className="form-header">
+    
+</div>
+
+<form >
+
+</form>
```
Comment (MaymoonaAlBoloshi) [inline src/pages/page1.tsx]: do we need these?

File: src/pages/page1.tsx
Comment (MaymoonaAlBoloshi) [inline src/pages/page1.tsx:1]: the steps should be in the in their own component, page one should contain only code for the first form

File: src/pages/page1.tsx (hunk 1 above)
Comment (ireemalqarni) [reply src/pages/page1.tsx]: No

File: src/pages/page1.tsx
Comment (ireemalqarni) [reply src/pages/page1.tsx:1]: changed done

**Pull request merged**